percona-xtradb-cluster-8.0-backup/run_backup.sh

```diff
@@ -78,16 +78,23 @@ backup_volume() {
 	vault_store $BACKUP_DIR/${SST_INFO_NAME}
 
 	if (($SST_FAILED == 0)); then
-		FIRST_RECEIVED=0
 		socat -u "$SOCAT_OPTS" stdio >xtrabackup.stream
-		FIRST_RECEIVED=1
 		if [[ $? -ne 0 ]]; then
 			log 'ERROR' 'Socat(2) failed'
 			log 'ERROR' 'Backup was finished unsuccessfully'
 			exit 1
 		fi
 		log 'INFO' "Socat(2) returned $?"
 	fi
+
+	trap '' 15
+	stat xtrabackup.stream
```

Comment:
Is this `stat` needed?

Reply:
it's been there for a long time but i don't think so, since the line below will fail if `xtrabackup.stream` does not exist

Reply:
I think we can do it